src/tezos-client/signers/remote.ts

```diff
@@ -1,10 +1,16 @@
 import { RemoteSigner } from "@taquito/remote-signer";
 import { readFile } from "fs/promises";
+import { join } from "path";
 import { REMOTE_SIGNER_CONFIG_FILE } from "src/utils/constants";
+import { globalCliOptions } from "src/cli/global";
```

Comment:
I am not happy about wiring `cli` globals here. We should for another way how to do this.